include/vstd/DateUtils.h

```diff
@@ -5,7 +5,7 @@ VCMI_LIB_NAMESPACE_BEGIN
 namespace vstd
 {
 
-	DLL_LINKAGE std::string getFormattedDateTime(std::time_t dt);
+	DLL_LINKAGE std::string getFormattedDateTime(std::time_t dt, std::string format = "");
```

Comment:
what's the use case of declaring default value and then copy-pasting code to read current format from settings? Either put that expression right here as default value (if possible) or create wrapper that passes it automatically (as I understand, that value can't be fetched inside a vstd function)

Reply:
It's not used yet. Could also remove default value if it's better?

I don't want to use settings in vstd. That doesn't make sense for a "std" file, I think.

Reply:
main point is to prevent copy-paste of reading from settings. You can create wrapper function inside the project (that is, not in vstd)

Reply:
Move to general text handler then?
Or to related TextOperations namespace?

Reply:
Yes, I just didn't think it was necessary at the moment, as it is currently only used three times (settings are already used two times). I also wouldn't know where such a wrapper would fit in well.

Reply:
well, 3 > 1 :)

I also don't know, try following Ivan's advice

Reply:
I suggest either TextOperations or general text handler class.
Even if there are "just" 3 usages, all 3 are rather hard to read due to extremely long statements.

So moving those into a dedicated method makes sense